### DISCUSSION TITLE
Noob Question: Is it possible to remove unwanted files from app > res

### DISCUSSION THREAD

**Discussion by kazimmt** [ReVanced/revanced-discussions#493] [Question] [answered]
When i download split apkm version, i extracted base apk. It has 3079 files inside Res Folder.
But when i download full apk, i extracted it. It has 9546 files inside Res folder.

We all know that, split apk also works on every device.
Is is possible to remove unwanted files from Res folder so the app will be less size compared to current version.

Or maybe a way to assemble revanced patches on base apk, so we can install it like other split apkm?

![Screenshot](https://user-images.githubusercontent.com/82371061/185046090-5b3a7310-dd46-4b14-93d6-ed54166fa595.jpg)

**Comment by TheJeterLP** [comment]
Why do you even care about the file size? Why should the team implement such a feature? In my opinion it's a waste of time.

**Comment by oSumAtrIX** [comment, marked as answer]
You can remove all libs from the full apk which are for different architectures than your phone's.

**Reply by kazimmt** [reply]
Ya, got it. Now it's only 65 mb. Thank you.
